### DISCUSSION TITLE
Websockets

### DISCUSSION THREAD

**Discussion by firstaxel** [coollabsio/coolify#1928] [Q&A] [answered]
Good evening I wanted to ask does coolify supports NextJS app with websockets

**Comment by amerkestijn** [comment]
Websockets are supported.

**Comment by firstaxel** [comment]
So I can deploy NextJS app with socket.io built inside

**Comment by amerkestijn** [comment, marked as answer]
Both websocket server and websocket client should work as i'm running my own server & clients.
If you run into any issues just let me know.
